hs_core/tests/api/rest/test_move_or_rename_file_or_folder.py

```diff
@@ -0,0 +1,85 @@
+import os
+import tempfile
+import zipfile
+
+from rest_framework import status
+
+from hs_core.hydroshare import resource
+from hs_core.tests.api.utils import MyTemporaryUploadedFile
+
+from .base import HSRESTTestCase
+
+
+class TestPublicUnzipEndpoint(HSRESTTestCase):

```

Comment:
It seems the test case here is to test rename_or_move, rather than unzip, so the class name TestPublicUnzipEndpoint may not be accurate here?